.github/workflows/pr-title-checker.yaml

```diff
@@ -0,0 +1,44 @@
+name: "PR Title Checker"
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+      - closed
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Run PR Title Checker
+        id: title-check
+        uses: thehanimo/pr-title-checker@v1.4.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          pass_on_octokit_error: false
```

Comment:
What should i do to set the githubtoken ?

Reply:
I think you don't need to do anything.I didn't  but it works @krishnaacharyaa

Reply:
It will set by the server or runner(according to github action) we don't need to worry about that sir @krishnaacharyaa